### DISCUSSION TITLE
How to trun a paint into texture

### DISCUSSION THREAD

**Discussion by aizcutei** [emilk/egui#4567] [Q&A] [answered]
Is there a way to turn a single paint like a `Frame` into a texture? I know there is a `screenshot` function but that gonna take the whole window in to a texture, I am wondering if there's a similar way to do so. I think this will be helpful for create frame buffer.

**Comment by YgorSouza** [comment, marked as answer]
No. egui doesn't have any rendering capabilities of its own (that is the goal of #1129 but it hasn't been implemented yet), and what it outputs to the backend ([FullOutput](https://docs.rs/egui/0.27.2/egui/output/struct.FullOutput.html)) does not allow distinguishing between the different UI elements. The best you can do at the moment is screenshot the whole window and then crop the part that you want, like the [save_plot](https://github.com/emilk/egui/blob/942fe4ab314323a66077fece4655c6e8e8c810a8/examples/save_plot/src/main.rs) example.
